.circleci/config.yml

```diff
@@ -1,4 +1,12 @@
-version: 2
+version: 2.1
+
+# Add additional CircleCI Orbs dependencies
+orbs:
```

Comment:
Hoorah! I do believe this is the first use of orbs! 👍

Reply:
Was good to avoid some wget binary + chmod scripting in favor of these.